test/core/test/each.test.ts

```diff
@@ -136,3 +129,36 @@ describe('context with each - concurrent', () => {
     })
   })
 })
+
+describe('not all arguments are array describe.each', () => {
+  const results = [null, [null]]
+  let i = 0
+
+  describe.each([null, [null]])('null is null', (value) => {
```

Comment:
maybe there are better names for these tests? there are 3 tests named `null is null`, but they check `[null] === [null]` 🤔